common/interpreter/wawaka_wasm/CMakeLists.txt

```diff
@@ -121,6 +120,14 @@ SET(SGX_INCLUDE_DIRS "${SGX_SDK}/include")
 LIST(APPEND SGX_INCLUDE_DIRS "${SGX_SDK}/include/tlibc")
 LIST(APPEND SGX_INCLUDE_DIRS "${SGX_SDK}/include/libcxx")
 
+#################################################################
+# WawakaCDI tool
+#################################################################
+
+IF (PDO_INTERPRETER STREQUAL "wawaka-aot")
+  ADD_SUBDIRECTORY(cdi)
+ENDIF()
+
```

Comment:
do we need to make a special case for this? we could just use a very simple "trust all" policy for everything but wawaka-aot and that would remove some of the special case situations.

Reply:
the `cdi` directory contains only the compilation report generation tool. I think having this package within the wawaka interpreter's directory hierarchy is causing some confusion. I'll move it elsewhere to make it clear that WawakaCDI has nothing to do with the interpreter.